libkbfs/tlf_journal.go

```diff
@@ -813,6 +826,56 @@ func (e errTLFJournalNotEmpty) Error() string {
 	return "tlfJournal is not empty"
 }
 
+func (j *tlfJournal) checkServerForConflicts(ctx context.Context) error {
+	if time.Since(j.lastServerMDCheck) < tlfJournalServerMDCheckInterval ||
+		!j.config.MDServer().IsConnected() {
+		return nil
+	}
+
+	isConflict, err := j.isOnConflictBranch()
+	if err != nil {
+		return err
+	}
+	if isConflict {
+		return nil
+	}
+
+	nextMDToFlush, err := func() (MetadataRevision, error) {
```

Comment:
You may need to replace this with `kbfsmd.Revision` if @akalin-keybase has already merged his refactor.